src/Life/Configuration.hs

```diff
@@ -142,9 +145,10 @@ resurrect CorpseConfiguration{..} = do
 
 -- TODO: should tomland one day support this?...
```

Comment:
Hmm. I guess we can create an issue in `tomland` for adding `Boolean` variable to print lists on each own line. And once this issue is implemented, we should be able to remove most of the logic from here. Do you think it's worth it to support such configuration in `tomland`?

Reply:
I've created: https://github.com/kowainik/tomland/issues/325